java/core/pom.xml

```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>com.lancedb</groupId>
         <artifactId>lance-parent</artifactId>
-        <version>0.0.5</version>
+        <version>0.18.1</version>
```

Comment:
Will these automatically get updated as part of the release process?  If not, we don't need to fix them in this PR but we should probably see if we can make that happen at some point.

Reply:
In Lance, only patch version got updated during release but not minor version. And the bump version is bump from existing version. That's why need to set it correct for once and for future minor version bump.
Lancedb has more solid version setting approach during release